### PR TITLE
[ZH] Update `How you can help!`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/How_you_can_help!/zh.md [hunk 1]
```diff
@@ -74,15 +69,15 @@ osu! 茁壮成长的社区也不乏技术力高超或竞技心火热的人，无
...
 ## 请避免负能量！让人们开心起来！
 
 古人有云：你应该成为解决问题的方案，而不是成为问题本身。只要能做到这点，你就是在帮助 osu! 社区变得更加积极向上！抱怨的人总会有的，但你仍然能用很多方式来点亮社区的心情：
 
-- 对彼此友善一些；生活本就苦短，争辩乃身外之物。
-- [举报你看到的作弊者](/wiki/Reporting_bad_behaviour/Handling_foul_play)，请不要通过其他渠道举报他们，例如使用 `!report <某人> 好像在作弊`举报，处理结果短则需要几分钟，多则可达数个小时。
-- 请给自己一点时间来细细冥想吧，把负能量中的消极不快筛除，只留下有助于事情解决的能量就好！
+- 对彼此友善一些。人生苦短，将其耗费在吵架上还是太浪费了。
+- [举报你看到的作弊玩家](/wiki/Reporting_bad_behaviour/Handling_foul_play)。不要填写模糊不清的备注，这无法在几分钟到几小时内解决你的问题，比如 `!report <某人> 好像在作弊`。
+- 尽量调解负能量，并将其转变为正能量。
```
Comment (CloneWith) [inline wiki/Community/How_you_can_help!/zh.md:81]: “调解”用在这里不是很合适（对双方/争论或纠纷），可以换成“消解”或“消减”

File: wiki/Community/How_you_can_help!/zh.md [hunk 2]
```diff
@@ -74,15 +69,15 @@ osu! 茁壮成长的社区也不乏技术力高超或竞技心火热的人，无
...
 
 ## 请避免负能量！让人们开心起来！
 
 古人有云：你应该成为解决问题的方案，而不是成为问题本身。只要能做到这点，你就是在帮助 osu! 社区变得更加积极向上！抱怨的人总会有的，但你仍然能用很多方式来点亮社区的心情：
 
-- 对彼此友善一些；生活本就苦短，争辩乃身外之物。
-- [举报你看到的作弊者](/wiki/Reporting_bad_behaviour/Handling_foul_play)，请不要通过其他渠道举报他们，例如使用 `!report <某人> 好像在作弊`举报，处理结果短则需要几分钟，多则可达数个小时。
-- 请给自己一点时间来细细冥想吧，把负能量中的消极不快筛除，只留下有助于事情解决的能量就好！
+- 对彼此友善一些。人生苦短，将其耗费在吵架上还是太浪费了。
+- [举报你看到的作弊玩家](/wiki/Reporting_bad_behaviour/Handling_foul_play)。不要填写模糊不清的备注，这无法在几分钟到几小时内解决你的问题，比如 `!report <某人> 好像在作弊`。
```
Comment (CloneWith) [inline wiki/Community/How_you_can_help!/zh.md:80]: 没看到原文有这样改，不过也没错？

```
这无法在几分钟到几小时内解决你的问题，比如 `!report <某人> 好像在作弊`。
```
这里有些奇怪，如果语意正确，可改成“这无法及时解决问题”。

---

原文 @ 20240329：
Please do not complain about them through other mediums as a simple `!report <username> seems to be cheating` can fix your problems within minutes to hours.

File: wiki/Community/How_you_can_help!/zh.md [hunk 3]
```diff
@@ -74,15 +69,15 @@ osu! 茁壮成长的社区也不乏技术力高超或竞技心火热的人，无
 
 从 2017 年开始，[osu! 团队](/wiki/People/osu!_team)的主要精力就转移到了 [osu!(lazer)](/wiki/Client/Release_stream/Lazer) 项目和新网站上，就是你现在看到的网站！你可以用很多方式让这些项目变得越来越好，让 osu! 逐渐靠近未来，以下是一部分建议：
 
-- 为 [osu!(lazer)](https://github.com/ppy/osu) 项目作出贡献。新版本的 osu! 有很多新东西需要加入，而你可以帮助实现这个目标！
+- 在 [osu!(lazer)](https://github.com/ppy/osu) 项目中作出贡献。新版本的 osu! 有很多新东西需要加入，而你可以帮助实现这个目标！
 - 让[网站](https://github.com/ppy/osu-web)越来越好。
 
 ## 请避免负能量！让人们开心起来！
 
 古人有云：你应该成为解决问题的方案，而不是成为问题本身。只要能做到这点，你就是在帮助 osu! 社区变得更加积极向上！抱怨的人总会有的，但你仍然能用很多方式来点亮社区的心情：
```
Comment (CloneWith) [inline wiki/Community/How_you_can_help!/zh.md:77]: “你应该成为问题解决方案的一部分”
“点亮社区的心情”，建议改成“改善...氛围”

**Review by CloneWith — APPROVED** [review]
一些段落开头的单空格缩进等等是之前已有的问题，这里先提出来了，可以改

File: wiki/Community/How_you_can_help!/zh.md [hunk 4]
```diff
@@ -35,36 +30,36 @@ osu! 有一个全天候监控着论坛和在线聊天的管理团队，他们致
...
+- 阅读一下[摸图指南](/wiki/Modding)和[谱面上架标准 (RC)](/wiki/Ranking_criteria) 来了解摸图的技术基础和上架谱面的必备要求。
+- 加入游戏聊天中的 `#mapping` 和 `#modreqs` 频道，参与讨论，或者看看别人在说什么。
+- 如果你有足够的时间和心力，你还可以提供一个[摸图列表](https://osu.ppy.sh/community/forums/60)——让谱师们发出让你摸图的请求的地方。
 
 ## 给玩家的进度或皮肤提供反馈！
 
 大家都乐于进步，并且时常需要游玩上的反馈。玩家们经常会在[游戏与排名板块](https://osu.ppy.sh/community/forums/13)征询自己游戏表现的反馈，还有人会在[制作中皮肤板块](https://osu.ppy.sh/community/forums/119)急切等待自己崭新皮肤的相关反馈。
 
-## 帮助或加入玩家自发组织的社区项目
+## 帮助或加入玩家自发组织的社区项目：
```
Comment (CloneWith) [inline wiki/Community/How_you_can_help!/zh.md:55]:
```suggestion
## 帮助或加入玩家自发组织的社区项目
```

File: wiki/Community/How_you_can_help!/zh.md [hunk 5]
```diff
@@ -35,36 +30,36 @@ osu! 有一个全天候监控着论坛和在线聊天的管理团队，他们致
...
-- 如果你有足够的时间和心力，你还可以提供一个[摸图列表](https://osu.ppy.sh/community/forums/60)，让谱师们请求你进行摸图的地方。
+- 找一找，或者在[待定 (Pending) 谱面列表](https://osu.ppy.sh/beatmapsets?s=pending)当中搜索你喜欢或愿意贡献的歌曲。
+- 玩一玩[*待定*或*制作中*](/wiki/Beatmap/Category#wip-and-pending)的谱面，并且在讨论页面给出你的反馈。
+- 阅读一下[摸图指南](/wiki/Modding)和[谱面上架标准 (RC)](/wiki/Ranking_criteria) 来了解摸图的技术基础和上架谱面的必备要求。
+- 加入游戏聊天中的 `#mapping` 和 `#modreqs` 频道，参与讨论，或者看看别人在说什么。
+- 如果你有足够的时间和心力，你还可以提供一个[摸图列表](https://osu.ppy.sh/community/forums/60)——让谱师们发出让你摸图的请求的地方。
 
 ## 给玩家的进度或皮肤提供反馈！
 
 大家都乐于进步，并且时常需要游玩上的反馈。玩家们经常会在[游戏与排名板块](https://osu.ppy.sh/community/forums/13)征询自己游戏表现的反馈，还有人会在[制作中皮肤板块](https://osu.ppy.sh/community/forums/119)急切等待自己崭新皮肤的相关反馈。
```
Comment (CloneWith) [inline wiki/Community/How_you_can_help!/zh.md:53]:
```suggestion
大家都乐于进步，并且时常需要反馈。玩家们经常会在[游戏与排名板块](https://osu.ppy.sh/community/forums/13)征询自己游戏表现的反馈，还有人会在[制作中皮肤板块](https://osu.ppy.sh/community/forums/119)急切等待关于自己新近制作皮肤的反馈。
```
“latest skin creation”，自己最近制作的皮肤

**Review by CloneWith — CHANGES_REQUESTED** [review]
点错了（

**Comment by TicClick** [comment]
hey, is there something else remaining to be done here?

**Comment by CloneWith** [comment]
There are some suggestions waiting for his opinions, either to be resolved or dismissed.

@YumeMuzi would you like to have a look on this?

**Comment by YumeMuzi** [comment]
> hey, is there something else remaining to be done here?

I'm not busy now, and it's time to push it

**Review by CloneWith — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**